src/Tizen.NUI.Components/Controls/Switch.cs

```diff
@@ -343,6 +339,8 @@ protected override void OnThemeChangedEvent(object sender, StyleManager.ThemeCha
 
         private void OnSelect()
         {
+            ((SwitchExtension)Extension)?.OnSelectedChanged(this);
```

Comment:
great! (b^_^)b

https://docs.microsoft.com/ko-kr/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-